etc/kayobe/kolla/config/octavia.conf

```diff
@@ -0,0 +1,21 @@
+[certificates]
+cert_generator = local_cert_generator
+ca_certificate = /etc/octavia/certs/server_ca.cert.pem
+ca_private_key = /etc/octavia/certs/server_ca.key.pem
+ca_private_key_passphrase = <server CA key passphrase>
+
+[health_manager]
+controller_ip_port_list = 10.208.1.0:5555, 10.208.1.1:5555, 10.208.1.2:5555
```

Comment:
Could we not template this... I think we have something similar for memcache here:
https://github.com/openstack/kolla-ansible/blob/e7a85726d23cdcf531fe0a82878d94314053cdfc/ansible/roles/octavia/templates/octavia.conf.j2#L36

Reply:
In fact I see the very line concerned there too;
https://github.com/openstack/kolla-ansible/blob/e7a85726d23cdcf531fe0a82878d94314053cdfc/ansible/roles/octavia/templates/octavia.conf.j2#L56

I think that the issue here is that we're using a new dedicated network 10.208/16 for the Octavia control and sync network. which will be unknown to kolla-ansible at the moment.

If the Octavia HMs bind instead to the internal API network and amphorae could route to them instead, then that could be a potential way around modifying kolla-ansible.

Reply:
Ah...

I suspect we need to add in octavia_interface into here somewhere:
https://github.com/openstack/kayobe/blob/aea7117764ee3ced0ccb7477c5c962569fa1c926/ansible/roles/kolla-ansible/defaults/main.yml#L104

But lets maybe just add a TODO to revisit that choice and fix it later.